src/components/expansion/ExpansionLogicEditor.svelte

```diff
@@ -34,14 +34,25 @@
     .getParsedAmpcsCommandDictionary(ruleDictionaryId, user)
     .then(parsedDictionary => (commandDictionaryJson = parsedDictionary));
 
-  $: if (monaco !== undefined && (commandDictionaryTsFiles !== undefined || activityTypeTsFiles !== undefined)) {
+  $: if (monaco !== undefined) {
     const { languages } = monaco;
     const { typescript } = languages;
     const { typescriptDefaults } = typescript;
     const options = typescriptDefaults.getCompilerOptions();
 
     typescriptDefaults.setCompilerOptions({ ...options, lib: ['esnext'], strictNullChecks: true });
-    typescriptDefaults.setExtraLibs([...commandDictionaryTsFiles, ...activityTypeTsFiles]);
+  }
+
+  $: if (monaco && commandDictionaryTsFiles) {
```

Comment:
We should also trigger changes off `activityTypeTsFiles` since we need to fire this if it changes.

Reply:
Actually on a closer read it should still fire because it's below?

Reply:
oops now it should work properly

Reply:
I combine the conditionals which makes it cleaner